build.gradle

```diff
@@ -17,6 +13,15 @@ buildscript {
         maven { url "https://jetbrains.bintray.com/kotlin-native-dependencies" }
         maven { url "https://plugins.gradle.org/m2/" }
     }
+    configurations.classpath {
```

Comment:
Please leave a comment why such resolution is required (because of 1.2-snapshot < 1.2.40?) for everything except native

Reply:
✔️